pkg/k8s/watchers/cilium_endpoint_slice.go

```diff
@@ -30,7 +61,8 @@ func (k *K8sWatcher) ciliumEndpointSliceInit(client *k8s.K8sCiliumClient, asyncC
 	cesNotify.Register(newCESSubscriber(k))
 
 	for {
-		_, cesInformer := informer.NewInformer(
+		// note: cesStore is has an index on node ips.
```

Comment:
```suggestion
		// note: cesStore has an index on node IPs.
```

But is this comment up-to-date? I cannot find any mention of a `cesStore` in the code